src/Common/src/Interop/Windows/mincore/Interop.Localization.cs

```diff
@@ -0,0 +1,121 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
```

Comment:
Agree. (I took the name from the Windows API set name: api-ms-win-core-localization.)